src/components/map/MapboxMap.tsx

```diff
@@ -68,7 +68,7 @@ const MapboxMap = ( {mapConfig, countriesConfig, studyPinsConfig}: MapboxMapProp
         ...mapConfig,
       };
 
-      const m = new mapboxgl.Map(mergedOptions);
+      const m = new mapboxgl.Map(mergedOptions).addControl(new mapboxgl.NavigationControl());
```

Comment:
This just adds nav controls to the map right?

Reply:
yep